components/ws-manager/pkg/manager/probe.go

```diff
@@ -42,7 +42,7 @@ type WorkspaceReadyProbe struct {
 
 // NewWorkspaceReadyProbe creates a new workspace probe
 func NewWorkspaceReadyProbe(workspaceID string, workspaceURL url.URL) WorkspaceReadyProbe {
-	workspaceURL.Path += "/_supervisor/v1/status/ide/wait/true"
+	workspaceURL.Path += "/_supervisor/v1/status/ide"
```

Comment:
@iQQBot what do you think about removing `wait/true`? I ask because you initially authored https://github.com/gitpod-io/gitpod/pull/13828. Is there anything you'd recommend we explicitly test?

I ask because @utam0k [did some testing here which is related](https://www.notion.so/gitpod/Why-does-gen75-s-workspace-start-slower-be5fc7fdb86c4ea7b964e1cb9cab4654#72d40f7f0f074141ab9b9f4fe8564fd4).

Reply:
I think the problem before was that ws-manager was only looking at HTTP status codes from the supervisor and didn't validate the body. I think this would increase the number of requests, is that the intent?